dragonfly/block/instrument/instrument.go

```diff
@@ -0,0 +1,88 @@
+package instrument
+
+// Instrument represents a note block instrument.
+type Instrument struct {
+	MagicNumber instrument
```

Comment:
Might want to unexport this field and expose a Uint8() method. People can still assign integers to this.